modules/message-lane/src/instant_payments.rs

```diff
@@ -46,10 +46,12 @@ where
 				Currency::transfer(submitter, relayer_fund_account, *fee, ExistenceRequirement::AllowDeath)
 					.map_err(Into::into)
 			}
-			Sender::Root | Sender::None => {
-				// fixme: we might want to add root account id to this struct.
-				Err("Root and None account is not allowed to send regular messages.")
+			Sender::Root => {
```

Comment:
@tomusdrw I had to change this to get Root messages to be sent across the bridge. Does this break what you had in mind when you added this check?

Reply:
Yes, it's not acceptable cause confirming such messages would fail (you would try to withdraw more from `relayer_fund_account` than it has - please see #559.

I'd suggest to ignore sending root messages for now.